.github/workflows/python-package.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
```

Comment:
name file : 
start-stack.yml

and here :
Start the stack

Reply:
Done!